docs/xks/developer-guide/scheduling-scaling.md

```diff
@@ -0,0 +1,254 @@
+---
+id: scheduling-scaling
+title: Scheduling and Scaling
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+The Kubernetes scheduler is responsible for assigning new Pods to Nodes. This functionality is a core component of Kubernetes. The Kuberetes Scheduler in its simplest form assigns Pods based on two main criteria.
+
+1. Filtering - Checks Nodes for available resources and if they meet the requirements of the Pod.
+2. Scoring - Ranks the filtered Nodes and chooses the highest ranked one.
+
+Most Kubernetes users encounter the scheduler for the first time when creating either a Deployment or Pod. The scheduler has to decide which Node to assign the Pod to. In the case of a Deployment the replica count can be increased and decreased at a moments notice by the user. When replica is changed for example from three to twenty the scheduler will have a lot of Pods all of a sudden to assign to Nodes. Having a deeper understanding of the Kubernetes scheduler will allow for a more efficient usage of resource consumption and debugging of any scheduler issues. A static replica count is the simplest method to work with Pod scaling and scheduling, this section will cover a lot of other options that exist in Kubernetes related to scaling and scheduling.
+
+## Pod Resources
+
+Setting the correct [resources for a Pod](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) is an important component in helping the scheduler select the correct Node to schedule a Pod to. There are two types of resource configuration fields, and these are configured for each container in a Pod.
+The resource request tells the scheduler how much of each resource each container in a Pod is expected to consume, it is however allowed of a Pod to consume more. The limit sets the maximum amount of resources each container in a Pod can consume. There are two main resource types that one should be aware of, CPU and memory.

```

Comment:
"... though a pod is allowed to consume more."

Reply:
Perhaps italics on "request" and "limit"?

Reply:
It may be good to mention that the pod is is likely to be killed if it exceeds the limit? That would underline the importance of limits.

Reply:
Limits is covered later in the chapter.